app/assets/src/components/utils/UrlQueryParser.js

```diff
@@ -10,7 +10,10 @@ class UrlQueryParser {
   }
 
   parse(query) {
-    let params = QueryString.parse(query);
+    let params = QueryString.parse(query, {
+      parseNumbers: false,
```

Comment:
According to the github repo, these values are actually the default (https://github.com/sindresorhus/query-string#parsenumbers)... why did you need this?

Reply:
Also, UrlQueryParser is not used below, right?

Reply:
Thanks for the catch regarding the defaults. Don't need to make a change in this file then.